dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -6,7 +6,7 @@ class Api::V1::UsersController < ApplicationController
 
   def load_user
     user_id = params[:user_id]
-    if user_id != 'me' && user_id != current_user.id
```

Comment:
The original use case only made use of `'me'` so the other part may have gone untested.